src/python/grpcio/grpc/_cython/_cygrpc/credentials.pyx.pxi

```diff
@@ -65,11 +66,50 @@ cdef int _get_metadata(void *state,
   return 0  # Asynchronous return
 
 
-cdef void _destroy(void *state) except * with gil:
-  cpython.Py_DECREF(<object>state)
+cdef extern from "<mutex>" namespace "std" nogil:
```

Comment:
I think we already included those: https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/_cython/_cygrpc/grpc.pxi#L45

Reply:
Thanks, fixed.